terraform/modules/application/security_groups.tf

```diff
@@ -0,0 +1,54 @@
+resource "aws_security_group" "ecs" {
+  name        = "${var.prefix}-ecs-security_group"
+  description = "ECS security group"
+  vpc_id      = var.vpc_id
+
+  lifecycle {
+    create_before_destroy = true
+  }
+}
+
+resource "aws_vpc_security_group_ingress_rule" "ecs_ingress" {

```

Comment:
N.B. Fine for the scope of this ticket, but should restrict this later once the load balancer is in play.

Reply:
Created ticket for this: https://dluhcdigital.atlassian.net/browse/CLDC-2710